collect_app/src/main/java/org/odk/collect/android/widgets/utilities/AudioPlayer.java

```diff
@@ -0,0 +1,20 @@
+package org.odk.collect.android.widgets.utilities;
+
+import org.odk.collect.audioclips.Clip;
+
+import java.util.function.Consumer;
+
+public interface AudioPlayer {
+
+    void play(Clip clip);
+
+    void pause();
+
+    void setPosition(String clipId, Integer position);
+
+    void onPlayingChanged(String clipID, Consumer<Boolean> playingConsumer);
```

Comment:
I wanted to avoid using `LiveData` inside `View` components as it means we need to pass them some kind of lifecycle to use for observations. This approach lets us use a more standard listener hook in the `View` but lets us use `LiveData` as the implementation so we have lifecycle awareness.